src/waveform/widgets/waveformwidgettype.h

```diff
@@ -23,10 +23,10 @@ class WaveformWidgetType {
         QtRGBWaveform,          // 15 RGB Qt
         GLSLRGBStackedWaveform, // 16 RGB Stacked
 #ifdef MIXXX_USE_QOPENGL
-        QOpenGLRGBWaveform,      // 17 RGB (QOpenGL)
-        QOpenGLLRRGBWaveform,    // 18 RGB (QOpenGL)
-        QOpenGLFilteredWaveform, // 19 Filtered (QOpenGL)
-        QOpenGLSimpleWaveform,   // 20 Simple (QOpenGL)
+        QOpenGLRGBWaveform,      // 17 RGB (all-shaders)
+        QOpenGLLRRGBWaveform,    // 18 L/R RGB (all-shaders)
+        QOpenGLFilteredWaveform, // 19 Filtered (all-shaders)
+        QOpenGLSimpleWaveform,   // 20 Simple (all-shaders)
 #endif
```

Comment:
I'm not sure I agree with the `(all-shaders)` naming, whats the reasoning for that? It doesn't really tell the user anything and neither does it tell anything to developers when users are filing waveform-related issues.

Reply:
Yeah, I am not convinced either, but (QOpenGL) was even less interesting.

The reasoning is that I want to express that these waveform types use a full stack of renderer layers that only use glsl shaders and no QPainter calls, which is what make them stand out.

I am very open to suggestions.

Reply:
Not sure, maybe just `(no QPainter)` would work I guess? I think discussing whats best for the user is futile anyways since they already have to decide what variant to choose out of the 4 different waveforms styles. It would best best if they didn't have to choose a variant, but thats out-of-scope. So the naming of one of those variants doesn't really matter IMO.